features/bevragen/persoon/verblijfplaats/dev/protocollering-vraag-gba.feature

```diff
@@ -0,0 +1,66 @@
+#language: nl
+
+Functionaliteit: protocollering van de vraag voor verblijfplaats
+
+  Rule: Met fields gevraagde velden worden geprotocolleerd als de elementnummers volgens Logisch ontwerp BRP
+    Dit is een 6-cijferige code, met zo nodig voorloopnul voor categorieën.
+    Ook alle velden die worden meegeleverd voor het bepalen van het verblijfplaats type worden vastgelegd in het veld 'request_zoek_rubrieken'.
+
+    Abstract Scenario: Met fields vragen om <fields> wordt vastgelegd als zoekrubrieken <zoek rubrieken>
+      Gegeven de persoon met burgerservicenummer '000000012' heeft de volgende gegevens
+      | naam            | waarde     |
+      | anummer (01.10) | 8468619643 |
```

Comment:
er is een index gezet op anummer. Dit betekent dat bestaande anummers niet kan worden gebruikt

Reply:
opgelost